rovercode/__init__.py

```diff
@@ -0,0 +1 @@
+"""Rovercode service application."""
```

Comment:
With Python 3 we technically don't need `__init__.py` in order to detect libraries...

Reply:
Interesting...

Reply:
https://stackoverflow.com/questions/37139786/is-init-py-not-required-for-packages-in-python-3